cheatsheets/Authentication_Cheat_Sheet.md

```diff
@@ -129,7 +129,7 @@ Some applications should use a second factor to check whether a user may perform
 
 #### TLS Client Authentication
 
-TLS Client Authentication, also known as two-way TLS authentication, consists of both, browser and server, sending their respective TLS certificates during the TLS handshake process. Just as you can validate the authenticity of a server by using the certificate and asking a verifiably-valid Certificate Authority (CA) if the certificate is valid, the server can authenticate the user by receiving a certificate from the client and validating against a third-party CA or its own CA. To do this, the server must provide the user with a certificate generated specifically for him, assigning values to the subject so that these can be used to determine what user the certificate should validate. The user installs the certificate on a browser and now uses it for the website.
+TLS Client Authentication, also known as two-way TLS authentication, consists of both browser and server sending their respective TLS certificates during the TLS handshake process. Just as you can validate the authenticity of a server by using the certificate and asking a verifiably-valid Certificate Authority (CA) if the certificate is valid, the server can authenticate the user by receiving a certificate from the client and validating against a third-party CA or its own CA. To do this, the server must provide the user with a certificate generated specifically for him, assigning values to the subject so that these can be used to determine what user the certificate should validate. The user installs the certificate on a browser and now uses it for the website.
```

Comment:
Nitpick:

> the server must provide the user with a certificate"

The certificate can come from another source …

Reply:
> The user installs the certificate on a browser and now uses it for the website.

Maybe it would be better for the state of clarity to say here that you actually install a *private key* alongside the certificate. People are often confused by this and think that a certificate is all that authenticates the user…

Maybe it would be nice to say that the private key can be shipped in the form of a smart card which is a lot more secure.

Maybe it would be interesting to add a comment saying that this is quite cumbersome to setup and that as a web application, you don't have a good control about the duration of the "session" when using such a method (logout is kinda meh).